tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreSampleAppsTest.cs

```diff
@@ -835,31 +837,32 @@ public void Builds_AzureBlazorWasmFunctionProject_By_Setting_Apptype_Via_BuildCo
                 () =>
                 {
                     Assert.True(result.IsSuccess);
-                    Assert.Contains(string.Format(SdkVersionMessageFormat, "3.1.402"), result.StdOut);
+                    Assert.Contains(string.Format(SdkVersionMessageFormat, DotNetCoreSdkVersions.DotNet80SdkVersion), result.StdOut);
                 },
                 result.GetDebugInfo());
         }
 
-        [Fact, Trait("category", "jamstack")]
-        public void Builds_AzureFunctionProject_FromBlazorFunctionRepo_When_Apptype_Is_SetAs_Functions()
+        [Theory]
+        [Trait("category", "jamstack")]
+        [InlineData("BlazorVanillaApiAppNet8")]
```

Comment:
May we also add `[InlineData("BlazorStarterAppNet8")]` to this test?

Reply:
Good catch, I forgot to add the starter app to this test -- just added it here and verified it passes locally.